storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -1293,6 +1302,10 @@ void Redo_Log_Data_Manager::track_archived_log(lsn_t start_lsn, const byte *buf,
     if (archived_log_monitor.get_reader().seek_logfile(
             reader.get_contiguous_lsn())) {
       archived_log_state = ARCHIVED_LOG_POSITIONED;
+    } else {
+#ifdef UNIV_DEBUG
+      ib::info() << "Failed to seek Archive log file from LSN " << start_lsn;
+#endif  // UNIV_DEBUG
```

Comment:
Should this msg be in release build? and failed to seek to log file should be WARNING level?